src/main/java/io/reactivex/Flowable.java

```diff
@@ -163,7 +163,7 @@ public static int bufferSize() {
      */
     @SchedulerSupport(SchedulerSupport.NONE)
     @BackpressureSupport(BackpressureKind.FULL)
-    public static <T, R> Flowable<R> combineLatest(Publisher<? extends T>[] sources, Function<? super T[], ? extends R> combiner) {
+    public static <T, R> Flowable<R> combineLatest(Publisher<? extends T>[] sources, Function<? super Object[], ? extends R> combiner) {
```

Comment:
It might be worth adding a sentence to the javadocs since people tend to ask why this isn't `? super T[]`

```
<p>
Note on method signature: since Java doesn't allow creating a generic array with `new T[]`, the implementation of this operator has to create an `Object[]` instead. Unfortunately, a `Function<Integer[], R>` passed in here would trigger a `ClassCastException`.
```

If you feel so, you could add these two sentences after the first sentence of all the affected methods and before the marble diagram (as people forget to scroll down below the diagram).